assets/src/edit-story/components/canvas/displayElement.js

```diff
@@ -80,7 +80,7 @@ function DisplayElement({ element }) {
         target.style.height = `${resize[1]}px`;
       }
       if (typeof dropTargets?.hover !== 'undefined') {
-        target.style.opacity = dropTargets.hover ? 0.6 : 1;
+        target.style.opacity = dropTargets.hover ? 0 : 1;
```

Comment:
I guess let's move with `0`, but I find it rather jarring. IMHO, we should do something like `0.1` instead. But it's a small nit: we can just go over it with @samitron7 on staging.

Reply:
Sg, last time I talked to sam we agreed on completely hiding it but we'll talk about it again tomorrow. Either way, should be in a separate PR